pvlib/pvsystem.py

```diff
@@ -1463,7 +1477,14 @@ def get_irradiance(self, solar_zenith, solar_azimuth, dni, ghi, dhi,
         poa_irradiance : DataFrame
             Column names are: ``'poa_global', 'poa_direct', 'poa_diffuse',
             'poa_sky_diffuse', 'poa_ground_diffuse'``.
+
+        See also
+        --------
+        :py:func:`pvlib.irradiance.get_total_irradiance`
```

Comment:
I think everything in `See also` is automatically linked to if possible, so no need for the rst markup